src/haddock/modules/_template_cat/README.md

```diff
@@ -4,25 +4,25 @@ To develop your own HADDOCK3 module follow the patterns used in other modules.
 
 1. If your module belongs to a new category, create a folder for that category
    under the `modules` folder.
-1. Else, create a folder for your module inside its relevant category.
+2. Else, create a folder for your module inside its relevant category.
```

Comment:
This is on-purpose, markdown will render it numerically independent of the sequence

Reply:
I know, but also it is not displayed properly in a simple text editor, while actual numeration looks good in both type of text editors